agreements/employment.md

```diff
@@ -0,0 +1,167 @@
+    4. You may carry over a maximum of five days' leave from one holiday year to the next.
+    5. At any time in a leave year, your accrued leave entitlement may be calculated as follows: 2.08 x Number of days worked in the year to date / 21.75
+    6. The amount of leave you may take will be the total leave accrued, rounded up to the nearest half a day, minus any leave you have already taken in the leave year.
+    7. We will usually permit you to take up to five days leave in advance of it being accrued.
+    8. If you wish to take leave, you must obtain approval (which will not be unreasonably withheld) and give notice of:
+        1. 28 days for a period of leave longer than one day;
+        2. seven days for a period of one day or less.
+    9. When your employment ends, you will receive payment in lieu for any part of your leave entitlement which has accrued but which has not been taken. Similarly, if you have taken more leave than you have accrued, we will deduct the appropriate amount of pay from any payment due from us to you on the termination of your employment or, if this is insufficient, we will require you to make up any shortfall.
+
+10. ### Sickness absence
+    1. dxw will pay your normal salary for up to ten days of sickness per year.
+    2. If you are frequently absent as a result of being sick, or are absent for a long period ("persistent sickness absence"), we may have no choice but to dismiss you.
+    3. dxw will not dismiss you for persistent sickness absence without making every reasonable effort to accommodate your circumstances. In particular, we will consider:
+        1. relevant medical information
+        2. options for alternative working arrangements or duties
+        3. your views and preferences
+    4. Further details of dxw's sickness absence arrangements are published in the Playbook.
+
+11. ### Parental leave
+    1. Details of dxw's parental leave arrangements are published in the Playbook.
+
+12. ### Pension
+    1. Details of dxw's pension arrangements are published in the Playbook.
+
+13. ### Expenses
+    1. We will reimburse you for out-of-pocket expenses that you incur in the course of your employment with us, subject to those expenses being reasonable, proportionate, necessary and evidenced.
+    2. Details of dxw's expenses policy and the process for claiming expenses are published in the Playbook.
+
+14. ### Retirement
+    1. We do not set any compulsory retirement age.
+
+15. ### Confidentiality
+    1. You will not use nor disclose to any other person any confidential information acquired by you during the course of your employment for any purpose other than the performance of your duties.
+    2. After your employment with us, any information that is sufficiently confidential as to amount to a trade secret must not be used by you in any future employment, or be disclosed to any other person, while that information retains its quality as a trade secret.
```

Comment:
What's a trade secret? Do I know any trade secrets?

Likewise, what is "confidential information"?